apps/condo/pages/ticket/create.tsx

```diff
@@ -6,27 +6,30 @@ import { useIntl } from '@core/next/intl'
 import { PageContent, PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
 import { OrganizationRequired } from '@condo/domains/organization/components/OrganizationRequired'
 import { ReturnBackHeaderAction } from '@condo/domains/common/components/HeaderActions'
+import { PageWithAuthBoundProps } from './[id]'
 
-const CreateTicketPage = () => {
+const CreateTicketPage = ({ AuthBound, TicketForm: TicketFormFromProps }) => {
     const intl = useIntl()
     const PageTitleMsg = intl.formatMessage({ id:'pages.condo.ticket.index.CreateTicketModalTitle' })
+    const ResAuthBound = AuthBound ? AuthBound : OrganizationRequired
```

Comment:
U need to encapsulate this logic onto component which will be used as wrapper on all pages.
Then you need to realise this if/else statement in thus component.
As the result you will isolate this repetitive logic.